docs/operations/server.md

```diff
@@ -11,140 +11,192 @@ describe how to configure JanusGraph Server to handle HTTP endpoint
+2142 [main] INFO  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - Configured application/vnd.graphbinary-v1.0 with org.apache.tinkerpop.gremlin.driver.ser.GraphBinaryMessageSerializerV1
+2142 [main] INFO  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - Configured application/vnd.graphbinary-v1.0-stringd with org.apache.tinkerpop.gremlin.driver.ser.GraphBinaryMessageSerializerV1
+2142 [main] WARN  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - The org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV3d0 serialization class is deprecated.
+2172 [main] INFO  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - Configured application/vnd.gremlin-v3.0+gryo with org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV3d0
+2172 [main] WARN  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - The org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV3d0 serialization class is deprecated.
+2172 [main] INFO  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - Configured application/vnd.gremlin-v3.0+gryo-stringd with org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV3d0
+2183 [main] INFO  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - Configured application/vnd.gremlin-v3.0+json with org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerV3d0
+2183 [main] INFO  org.apache.tinkerpop.gremlin.server.AbstractChannelizer  - Configured application/json with org.apache.tinkerpop.gremlin.driver.ser.GraphSONMessageSerializerV3d0
+2240 [gremlin-server-boss-1] INFO  org.apache.tinkerpop.gremlin.server.GremlinServer  - Gremlin Server configured with worker thread pool of 1, gremlin pool of 8 and boss thread pool of 1.
+2240 [gremlin-server-boss-1] INFO  org.apache.tinkerpop.gremlin.server.GremlinServer  - Channel started at port 8182.
+```
+
+
+JanusGraph Server is configured by the provided YAML file `conf/gremlin-server/gremlin-server.yaml`. 
+That file tells JanusGraph Server many things and is based on the Gremlin Server config, see [Gremlin Server](https://tinkerpop.apache.org/docs/current/reference/#gremlin-server).
+
+### Usage of janusgraph-server.sh
+
+The JanusGraph Server can be started as foreground with stdout logging or detached. 
+
+```txt
+$ ./bin/janusgraph-server.sh
+Usage: ./bin/janusgraph-server.sh {start|stop|restart|status|console|usage <group> <artifact> <version>|<conf file>}
+
+    start           Start the server in the background using conf/gremlin-server/gremlin-server.yaml as the
+                    default configuration file
+    stop            Stop the server
+    restart         Stop and start the server
+    status          Check if the server is running using the pid file
+    console         Start the server in the foreground using conf/gremlin-server/gremlin-server.yaml as the
+                    default configuration file
```

Comment:
`usage` should now also be explained here, given that you added it to the `janusgraph-server.sh` script, right?